tests/test_head_loss.py

```diff
@@ -487,6 +496,10 @@ def energy_system_options(self):
                 network_type=NetworkSettings.NETWORK_TYPE_GAS,
                 pressure=solution.parameters(0)[f"{pipes[0]}.pressure"],
             )
+            # Compare the hydraulic power to hard-coded values. Difference expected if an error
+            # occours in the calculation of the gas kinematic viscosity.
+            if head_loss_option_setting == HeadLossOption.LINEARIZED_ONE_LINE_EQUALITY:
```

Comment:
The reason for this is because we are iterating over several head loss options, and it is only necessary to do this check for one of the head loss options.